daemon/inertiad/project/deployment.go

```diff
@@ -182,12 +183,29 @@ func (d *Deployment) Deploy(
 		fmt.Fprintln(out, "Continuing...")
 	}
 
+	// Send build start slack notification
+	notification := notifier.NewNotifier()
+	notifyErr := notification.Notify("Build started")
+	if notifyErr != nil {
+		return func() error { return nil }, notifyErr
+	}
+
 	// Build project
 	deploy, err := d.builder.Build(strings.ToLower(d.buildType), *conf, cli, out)
 	if err != nil {
+		notifyErr = notification.Notify("Build error")
+		if notifyErr != nil {
+			fmt.Println(out, notifyErr)
```

Comment:
`out` is an `io.Writer` - if you want to write to it, you need to use `FPrint`, like:

```suggestion
			fmt.Fprintln(out, notifyErr)
```

Reply:
you can also write this in one line like htis:

```go
if notifyErr := notification.Notify("Build error");  notifyErr != nil {
   ...
}
```

same for your other usages of `notifyErr`